lib/application/widgets/theme/update_memo_terminal.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/material.dart';
```

Comment:
> I wasn't aiming for abstraction, but I agree with your point. What about rename to UpdateMemoTerminal?

Sure, but I would move it to the respective folder, as it doesn't make sense to be used other in that particular context.